src/constant/index.js

```diff
@@ -540,6 +540,45 @@ const ERRORS = [
     description:
       "Displayed on the attempt of logging in. All seats within a given license are already taken.",
   },
+  {
+    id: "agent_offline",
+    message: "Agent offline",
+    description:
+      "Request couldn't be performed when agent is offline.",
```

Comment:
That is not correct - the message is returned by a server reacting to a request, so the request has already been performed - it just couldn't be processed/completed by the server. 
Also I'm not sure about mixing the tenses in this message - @ale-kacperczyk WDYT?

Reply:
So something along the lines of `The server couldn't process the request because the agent was offline.`?

Good point with mixing tenses.